.travis/build.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+if [[ $TRAVIS_OS_NAME == 'osx' ]]; then
```

Comment:
Travis never seems to get here. Travis is effectively only building the library on Linux and never testing it.

Reply:
Fixed in #66.